src/components/News.jsx

```diff
@@ -32,6 +32,15 @@ class News extends React.Component {
           pageNum * this.props.itemsPerPage + this.props.itemsPerPage
         )
         .map(item => {
+          let cutarr = [' ',',','.','\n'];
+          if(item.node.content.content.length > 400){
+            for(let i = 400; i < item.node.content.content.length; i++){
+              if(cutarr.indexOf(item.node.content.content[i]) > -1){
```

Comment:
you could probably use a regex to check for whitespace characters, like this
`/\s/.test(item.node.content.content[i])`